bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/impl/Strategy.scala

```diff
@@ -86,7 +86,7 @@ object Strategy {
         ao.get.asInstanceOf[InstAltair].getMode match {
           case AltairParams.Mode.LGS_P1 => List(Pwfs1North)
           case AltairParams.Mode.LGS_OI => List(GmosNorthOiwfs)
-          case _                        => List(AltairAowfs, Pwfs1North, GmosNorthOiwfs)
```

Comment:
By the info in REL-2323, I don't think that that is what we want. For all combinations other than those listed, we want only Altair and Altair + GMOS OI. Since we don't want GMOS OI by itself, we can't include it in this list; however, the probe handling changes I made to `AltairParams.Mode` takes care of this and provides the option to add the appropriate guide stars.